kevm-pyk/src/kevm_pyk/kproj/evm-semantics/lemmas/evm-int-simplification.k

```diff
@@ -20,36 +20,36 @@ module EVM-INT-SIMPLIFICATION-COMMON
     rule [upInt-lt-true]:
       ((X up/Int Y) *Int Y) <Int Z => true
       requires X +Int Y <=Int Z andBool 0 <Int Y
-      [simplification]
+      [simplification, preserves-definedness]
 
     rule [upInt-lt-false]:
       ((X up/Int Y) *Int Y) <Int Z => false
       requires Z <=Int X andBool 0 <Int Y
-      [simplification]
+      [simplification, preserves-definedness]
```

Comment:
Same here, it looks like every symbol is total?

Reply:
`A up/Int B` is not defined for `B ==0`. 
(and for clarification: For equations, booster checks the LHS for partiality, too, to avoid evaluating undefined things to defined ones - even though many other parts of the code assume the LHS to always be defined)

Reply:
> `A up/Int B` is not defined for `B ==0`. 

After double-checking this I find that `up/Int` is actually a total function (`0` if divisor `<= 0`). Only the rules with `modInt` need the additional attribute.